Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -606,18 +606,21 @@ bool SVertexer::checkV0(const TrackCand& seedP, const TrackCand& seedN, int iP,
 
     // Setup looser cuts for the DCAFitter
     fitterV0.setMaxDZIni(mSVParams->mTPCTrackMaxDZIni);
-    fitterV0.setMaxDXYIni(mSVParams->mTPCTrackMaxDXYIni);
+    // fitterV0.setMaxDXYIni(mSVParams->mTPCTrackMaxDXYIni);
```

Comment:
What is the reason for suppressing this cut?

Reply:
No reason, slipped through the cracks when testing, thanks for catching.